truffle/com.oracle.truffle.api.debug/src/com/oracle/truffle/api/debug/ExecutionEvent.java

```diff
@@ -87,9 +90,11 @@ public Debugger getDebugger() {
      * </ul>
      * 
      * @since 0.9
+     * @deprecated Continue is by default, this method is not necessary.
      */
```

Comment:
I don't think it is necessary to deprecated this method. I know continue is the default action, but having the method allows people to "change their mind". It keeps some symmetry and also advertises that debugger can "continue".

I suggest to integrate the change without this deprecation.